_data/sites.json

```diff
@@ -3682,9 +3682,9 @@
     {
         "meta": "popular",
         "name": "eBay",
-        "url": "https://ocsnext.ebay.com/ocs/ca",
-        "difficulty": "hard",
-        "notes": "A few survey questions will be asked prior to account deletion.",
+        "url": "https://accountsettings.ebay.ca/uas",
+        "difficulty": "medium",
+        "notes": "Hover over name on top left, click 'Account settings', replace all of your personal info with fake info before closing the account, click 'Close account' under the Account preferences column, then scroll down and click 'Close account and delete my data'. Select why you are leaving from drop down and continue. Click on checkmark and then click the left button to close your account.",
```

Comment:
Why changing the data if it's deleted? Just to be safe?

Reply:
Yes exactly. This should go without saying for privacy aware users.